docs/codeql/codeql-language-guides/customizing-library-models-for-csharp.rst

```diff
@@ -340,3 +340,19 @@ The first four values identify the callable (in this case the getter of the ``No
 - The fourth value ``()`` is the method input type signature.
 - The fifth value ``summary`` is the kind of the neutral.
 - The sixth value ``manual`` is the provenance of the neutral.
+
+.. _threat-models-csharp:
+
+Threat models
+-------------
+
+.. include:: ../reusables/beta-note-threat-models.rst
+
+A threat model is a named class of dataflow sources that can be enabled or disabled independently. Threat models allow you to control the set of dataflow sources that you want to consider unsafe. For example, one codebase may only consider remote HTTP requests to be tainted, whereas another may also consider data from local files to be unsafe. You can use threat models to ensure that the relevant taint sources are used in a CodeQL analysis.
+
+The ``kind`` property of the ``sourceModel`` determines which threat model a source is associated with. There are two main categories:
```

Comment:
```suggestion
The ``kind`` property of ``sourceModel`` determines which threat model a source is associated with. There are two main categories:
```

Probably debatable but generally I wouldn't expect to see the article before (for example) `sourceModel`. Very minor, in any case!